src/layer.rs

```diff
@@ -330,7 +332,7 @@ impl Layer {
     /// [`WriteOptions`] serialization format configuration.
     ///
     /// The path should not exist.
-    pub fn save_with_options(&self, path: &Path, opts: &WriteOptions) -> Result<(), Error> {
+    pub(crate) fn save_with_options(&self, path: &Path, opts: &WriteOptions) -> Result<(), Error> {
```

Comment:
We are using save_with_options in ufofmt.  I think you do want this pub so users can modify their write formats?

Reply:
are you using it on `Layer`, explicitly? My thinking was that it should be public at the `Font` level, but not for individual layers.

Reply:
Apologies.  I shouldn't review on the mobile app... :)

Only using with Font. Thanks!